### PR TITLE
[VarDumper] Relax 1 test failing with latest PHP versions

### PR DESCRIPTION
**Pull request by remicollet**
| Q | A |
| --- | --- |
| Branch? | 2.8 |
| Bug fix? | no |
| New feature? | o |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | N/A |
| License | MIT |
| Doc PR | reference to the documentation PR, if any |

Related to php bug https://bugs.php.net/72646 which is fixed in 5.6.25RC1, 7.0.10RC1, 7.1.0beta2

Detected in Fedora CI, failed since 7.0.10RC1, see
https://apps.fedoraproject.org/koschei/package/php-symfony

### CODE REVIEW

File: src/Symfony/Component/VarDumper/Tests/Caster/SplCasterTest.php [hunk 1]
```diff
@@ -94,10 +94,8 @@ public function testCastFileObject()
   file: true
   dir: false
   link: false
-%AcsvControl: array:2 [
-    0 => ","
-    1 => """
```
Comment (nicolas-grekas) [inline src/Symfony/Component/VarDumper/Tests/Caster/SplCasterTest.php]: can't we keep these two first lines and add %A for the third one?

Comment (remicollet) [inline src/Symfony/Component/VarDumper/Tests/Caster/SplCasterTest.php]: sure ;)
But does it really worth it ?.

Comment (remicollet) [inline src/Symfony/Component/VarDumper/Tests/Caster/SplCasterTest.php]: done

**Comment by nicolas-grekas** [comment]
@remicollet can you remove the first commit please?

**Comment by remicollet** [comment]
Sorry, I don't how.... :(
But as already in 2.8, should be OK (see the "This branch has no conflicts with the base branch"), else download the patch and use git am ;)   (Damned github...)

**Comment by nicolas-grekas** [comment]
Thank you @remicollet.

**Pull request merged**